modules/op-configuring-pipelines-as-code-logging.adoc

```diff
@@ -0,0 +1,151 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-logging_{context}"]
+= Configuring {pac} logging 
+
+You can configure {pac} logging by editing the `pac-config-logging` config map in the `TektonConfig` custom resource (CR).
+
+.Prerequisites
+
+* {pac} is installed on your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Administration* → *CustomResourceDefinitions*.
+
+. Use the *Search by name* box to search for the `tektonconfigs.operator.tekton.dev` custom resource definition (CRD) and click *TektonConfig* to see the CRD details page.
+
+. Click the *Instances* tab.
+
+. Click the *config* instance to see the `TektonConfig` CR details.
```

Comment:
What is the instance? Is it a link that shows a pop up window? If so, I think it should be
```suggestion
. Click *config*  to view the `TektonConfig` CR details in a pop-up window.
```
See https://redhat-documentation.github.io/supplementary-style-guide/#popup for more

Reply:
I am pretty sure there is not a pop-up window, when the user clicks on the *config* instance, but I will double check.

Reply:
The instance is the config. That is the name for that particular instance.